src/EnsembleKalmanInversions.jl

```diff
@@ -41,6 +42,8 @@ mutable struct EnsembleKalmanInversion{E, I, M, O, S, R, X, G, C, F}
     unconstrained_parameters :: X
     forward_map_output :: G
     pseudo_stepping :: C
+    precomputed_arrays :: P
```

Comment:
What are "precomputed_arrays"?

Reply:
I agree that we should move the fixes to a new PR.

Reply:
It's all developer stuff; it doesn't affect any of the user scripts.